samples/drivers/uart/single_wire/boards/stm32f3_disco.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 Jonathan Hahn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	 aliases {
+		single-line-uart1 = &usart2;
+		single-line-uart2 = &uart4;
+	};
+};
+
+&usart2 {
+	single-wire;
+};
```

Comment:
Can't we also update pinctrl-0 here and configure only one pin ?

Reply:
Never changed anything there, but I'll look into that.

Reply:
Updated binding description makes this mandatory.

Reply:
Done